conf/general-example.ubuntu

```diff
@@ -20,6 +20,9 @@ define ("COOKIEDOMAIN", "dev.openaustralia.org");
 
 define ("CONTACTEMAIL", "contact@openaustralia.org");
 
+// File system path to the html email tamplate
+define ("HTML_EMAIL_TEMPLATE","/srv/www/openaustralia/twfy/www/includes/easyparliament/templates/emails/html_email_template.html");
```

Comment:
Since the templates are always in the code repository at the same location, I don't think this needs to be configurable.